Reorganize certificate config intro page (#2978)
zowe/docs-site: docs/user-guide/use-certificates.md

# Use certificates

> Once you have generated or imported your certificates you can now use the certificates with Zowe. Use the procedure that corresponds to the type of certificates you generated or imported: 

Reviewer: ```suggestion
Once you have generated or imported your certificates, you can now use the certificates with Zowe. Use the procedure that corresponds to the type of certificates you generated or imported: 
```

- [Use PKCS12 certificates](#use-pkcs12-certificates)
- [Use JCERACFKS certificates](#use-jceracfks-certificates)

## Use PKCS12 certificates

To use PKCS12 certificates, run the command `zwe start -c ./zowe.yaml` in the directory with the `zowe.yaml` file to start Zowe.

> When Zowe is launched, details for the PKCS12 certificate used are specified in the `zowe.yaml` section `certificates`. This section contains information about the certificate name and the location of the certificate, together with the truststore location.

Reviewer 2: ```suggestion
The details about the PKCS12 certificate used when Zowe is launched, are specified in the `zowe.yaml` section `certificates`. This section contains information about the certificate name and the location of the certificate, together with the truststore location.
```

The two most common scenarios for using a PKCS12 certificate are:  
* You have an existing certificate and wish to configure Zowe to use the certificate.
* You do not have a certificate and wish to [generate a new certificate](./generate-certificates.md).  

  The `zwe init certificate` command supports both scenarios. The input parameters that control certificate configuration are specified in the section `zowe.setup.certificates`.

> To check startup of Zowe, see [Troubleshooting startup of Zowe z/OS components](https://docs.zowe.org/stable/troubleshoot/troubleshoot-zos-startup).

Reviewer 2: ```suggestion
To troubleshoot issues during Zowe startup, see [Troubleshooting startup of Zowe z/OS components](https://docs.zowe.org/stable/troubleshoot/troubleshoot-zos-startup).
```

## Use JCERACFKS certificates

> When Zowe is launched, details for the JCERACFKS certificate used are specified in the `zowe.yaml` section `certificates`.  This contains information for the certificate name and its location, together with the truststore location.  

Reviewer 2: ```suggestion
The details about the JCERACFKS certificate used when Zowe is launched, are specified in the `zowe.yaml` section `certificates`.  This contains information about the certificate name and its location, together with the truststore location.  
```

    - "safkeyring:////stcusername/KeyName&ca name 2"
```

> If you run into the error message `<ZWED:527259> SZOWE CRITICAL unable to get issuer certificate`, check this section in your `zowe.yaml` file and verify that the syntax is correct.

Reviewer 3: SZOWE ? Is that really correct?

> If you receive the error message, `<ZWED:527259> SZOWE CRITICAL unable to get issuer certificate`, check this section in your `zowe.yaml` file and verify that the syntax is correct.

Reviewer: ```suggestion
If you receive the error message, `<ZWED:527259> ZOWE CRITICAL unable to get issuer certificate`, check this section in your `zowe.yaml` file and verify that the syntax is correct.
```
